src/coreclr/vm/eventreporter.cpp

```diff
@@ -110,7 +110,7 @@ EventReporter::EventReporter(EventReporterType type)
 
     case ERT_ManagedFailFast:
         if(!ssMessage.LoadResource(CCompRC::Optional, IDS_ER_MANAGEDFAILFAST))
-            m_Description.Append(W("Description: The application requested process termination through Environment.FailFast."));
+            m_Description.Append(W("Description: The application requested process termination through System.Environment.FailFast(string message)."));
```

Comment:
`Environment.FailFast` has multiple overloads. I do not think that this change is an improvement.

Reply:
Is the resource version wrong then?

Reply:
Ah ok. I see that you have just copied over the string from resources.

Reply:
I would delete "(string message)" from both here and the resource.

Reply:
Yeah, we assumed it was just forgotten during another reflector. 

Will do!